### PR TITLE
Docs: Fix line break issue in Schemas and Content Models section

### PR DESCRIPTION
**Pull request by imanish003**
## What is the issue?
There is a line break issue in [this documentation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-transforms/#schemas-and-content-models) as shown in the screenshot below:
image

## What this PR does?
This should fix the line break issues shown in the screenshot below.

### CODE REVIEW

File: docs/reference-guides/block-api/block-transforms.md [hunk 1]
```diff
@@ -223,18 +223,9 @@ transforms: {
...
-were copied.
-
-When writing `raw` transforms you can control this by supplying a `schema` which describes allowable content and which
-will be applied to clean up the pasted content before attempting to match with your block. The schemas are passed
-into [`cleanNodeList` from `@wordpress/dom`](https://github.com/wordpress/gutenberg/blob/trunk/packages/dom/src/dom/clean-node-list.js); check there for
-a [complete description of the schema](https://github.com/wordpress/gutenberg/blob/trunk/packages/dom/src/phrasing-content.js).
+When pasting content it's possible to define a [content model](https://html.spec.whatwg.org/multipage/dom.html#content-models) that will be used to validate and process pasted content. It's often the case that HTML pasted into the editor will contain a mixture of elements that _ should_ transfer as well as elements that _shouldn't_. For example, consider pasting `<span class="time">12:04 pm</span>` into the editor. We want to copy `12:04 pm` and omit the `<span>` and its `class` attribute because those won't carry the same meaning or structure as they originally did from where they were copied.
```
Comment (skorasaurus) [inline docs/reference-guides/block-api/block-transforms.md]: thanks for the fix and sorry for the delay: ; looks great and ready to be merged except could you fix the following: `_should_` ; (no space between `_` and `should`)

**Review by skorasaurus — CHANGES_REQUESTED** [review]

File: docs/reference-guides/block-api/block-transforms.md (hunk 1 above)
Comment (skorasaurus) [reply docs/reference-guides/block-api/block-transforms.md]: otherwise, you should be fine; there have been no edits to the file since your PR.

Comment (imanish003) [reply docs/reference-guides/block-api/block-transforms.md]: Thanks for the review 🙌🏻 I have removed the space between _ and should in d7bf196. 🙂

**Review by skorasaurus — APPROVED** [review]
Thank you again for the improvement; looks good to me.

**Comment by imanish003** [comment]
@skorasaurus Thanks 🙌🏻 May I know who will be merging this PR? 🙂

**Review by t-hamano — APPROVED** [review]
LGTM 👍

Below is a screenshot of this document on GitHub, but since this is a markdown file, normal line breaks are not recognized as line breaks and are displayed correctly.

However, when it was synced to the block editor handbook, this line break was considered a `br` tag, which might have caused unintended consequences.

Thank you for your great work!

md

**Comment by imanish003** [comment]
@t-hamano Thanks for the explanation 🙌🏻

> However, when it was synced to the block editor handbook, this line break was considered a br tag, which might have caused unintended consequences.

Understood. Given that this line break was interpreted as a br tag in the block editor handbook, it's possible that there could be more such unintended consequences. Do you think it would be a good idea to have someone review all the relevant documentation to ensure there are no further issues? 🙂

**Comment by t-hamano** [comment]
@imanish003 

Thanks to your PR, I can confirm that there are many such cases.

However, some documentation is generated from JSDoc comments by the `docgen` package. As shown below, the portion surrounded by tokens is automatically generated from JSDoc, so the markdown file can not be updated directly.

```md
<!-- START TOKEN ... -->

Auto-generated text from JSDoc would go here

<!-- END TOKEN ... -->
```

This case will be fixed in #49635 I put up.

For content outside of this token in the markdown file, it should be possible to correct the unnatural line breaks. I would like to put this issue on my task list, but PR from you is also welcome 🙌

**Comment by imanish003** [comment]
@t-hamano Thank you for providing a clear explanation! I'm glad to know that the issue is being addressed in #49635. Thanks again for quickly taking care of this 🙌🏻

**Pull request merged**